tests/publiccloud/register_system.pm

```diff
@@ -37,7 +37,11 @@ sub run {
         $arch = "aarch64" if ($arch eq "arm64");
         for my $addon (@addons) {
             next if ($addon =~ /^\s+$/);
-            if (is_sle('<15') && $addon =~ /tcm|wsm|contm|asmm|pcm/) {
+            record_info $addon, "Going to register '$addon' addon";
+            if ($addon =~ /ltss/) {
+                my $regcode = get_var('SCC_REGCODE_LTSS');
```

Comment:
I think that's one of the cases, where we should use `get_required_var` instead.

Reply:
Line 42, Don't know why the other line was marked

Reply:
Check now please

Reply:
no, LTSS is not always thre ...

Reply:
But this gets triggered only when `ltss` is part of `SCC_ADDONS` @dzedro

Reply:
With `get_required_var` you will break non-LTSS versions

Reply:
But it will be only required for `$addon =~ /ltss/`. And I think there it should fail, if there is not LTSS Regcode defined, right?

Reply:
Exactly

Reply:
Can I then keep it like that?